deploy/cephcsi/image/Dockerfile

```diff
@@ -1,15 +1,42 @@
 FROM ceph/ceph:v14.2
-LABEL maintainers="Ceph-CSI Authors"
-LABEL description="Ceph-CSI Plugin"
 
-# To support cross building, do NOT RUN native binaries here.
-# If we have to run native binaries, qemu-user-static must be installed on
-# build host and mounted to container.
+LABEL stage="build"
 
-# Removing ceph-iscsi repo to workaround the repo issue while upgrading
-#RUN rm -f /etc/yum.repos.d/ceph-iscsi.repo && yum -y update && yum clean all
-ENV CSIBIN=/usr/local/bin/cephcsi
+ARG GOLANG_VERSION
+ARG GOARCH
+ARG CSI_IMAGE_NAME
+ARG GIT_COMMIT
+ARG CSI_IMAGE_VERSION
 
-COPY cephcsi $CSIBIN
+RUN yum install wget libcephfs-devel librados-devel librbd-devel /usr/bin/cc /usr/bin/c++ make -y
+RUN wget https://dl.google.com/go/go${GOLANG_VERSION:-1.13.8}.linux-amd64.tar.gz && tar xzf go${GOLANG_VERSION:-1.13.8}.linux-amd64.tar.gz -C /usr/local/go
```

Comment:
this will not work. you can not cross-compile when cgo is used (required for go-ceph).